docs/resources/aws_transit_gateway_routes.md

```diff
@@ -0,0 +1,141 @@
+---
+title: About the aws_transit_gateway_routes Resource
+platform: aws
+---
+
+# aws\_transit\_gateway\_routes
+
+Use the `aws_transit_gateway_routes` InSpec audit resource to test the properties of all Route for a AWS transit gateway route table.
+To audit a single gateway route, use the `aws_transit_gateway_route` (singular) resource.
+
+## Syntax
+
+An `aws_transit_gateway_routes` resource block collects a group of transit gateway routes' descriptions and then tests that group.
+
+    describe aws_transit_gateway_route(transit_gateway_route_table_id: 'tgw-rtb-08acd74550c99e589')
+      it { should exist }
+    end
+
+### Parameters
+
+The AWS transit gateway route table ID is required.
```

Comment:
`transit_gateway_route_table_id` _(required)_ 
The documentation guy told me to write like this.

Reply:
you mean like [this](https://github.com/inspec/inspec-aws/pull/355/files#diff-68c91cb0b505b8f85d32c6b4f300c908712dcbe7f71f8db7e5319302ed14fdf3R30)